src/test/java/htsjdk/samtools/SAMRecordUtilTest.java

```diff
@@ -24,7 +24,7 @@
     @Test public void testSafeReverseComplement() throws CloneNotSupportedException {
         final SAMRecord original = createTestSamRec();
         final SAMRecord cloneOfOriginal = (SAMRecord) original.clone();
-        //Runs a safe reverseComplement
+        //Runs a copy (rather than in-place) reverseComplement
         SAMRecordUtil.reverseComplement(cloneOfOriginal, Arrays.asList("Y1"), Arrays.asList("X1", "X2", "X3", "X4", "X5"), false);
 
         Assert.assertEquals(original.getReadString(), "ACACACACAC");
```

Comment:
could you expand this test using a data-provider to show that if false->true then the assertion failes?